crates/ntt/src/multithreaded.rs

```diff
@@ -144,7 +157,7 @@ fn forward_transform<F: BinaryField, P: PackedField<Scalar = F>>(
 			.into_par_strides(1 << log_stride_len)
 			.for_each(|mut stride| {
 				// i indexes the layer of the NTT network, also the binary subspace.
-				for i in (0..par_rounds).rev() {
+				for i in (0..par_rounds.saturating_sub(skip_rounds)).rev() {
 					let s_evals_par_i = &s_evals[log_y - par_rounds + i];
```

Comment:
hmm weird @jimpo i am getting `attempt to subtract with overflow` here during the test `tests::test_boundaries`. no other test triggers it, and it appears to only trigger in the multithreaded case... any thoughts?